worker/domainVerification.js

```diff
@@ -0,0 +1,273 @@
+import createPrisma from '@/lib/create-prisma'
+import { verifyDNSRecord, issueDomainCertificate, checkCertificateStatus, getValidationValues, deleteDomainCertificate } from '@/lib/domain-verification'
+import { datePivot } from '@/lib/time'
+
+const VERIFICATION_INTERVAL = 60 * 5 // 5 minutes
```

Comment:
I'm pretty sure you and @ekzyis debated this, but I think every 5 minutes is too slow.

My expectation as a user is that after I set DNS, within ~30-60 seconds I see a success message.

Ideally, it might be best to backoff to 5 minutes after an hour, but before an hour has passed do every 30 seconds.